benefits/core/admin/common.py

```diff
@@ -1,8 +1,31 @@
 from django.contrib import admin
+from django.http import HttpRequest
 
 from benefits.core import models
 
 
 @admin.register(models.PemData)
 class PemDataAdmin(admin.ModelAdmin):
     list_display = ("label",)
+
+    def get_exclude(self, request, obj=None):
+        fields = []
+
+        if not request.user.is_superuser:
+            fields.extend(["remote_url", "text_secret_name"])
+
+        return fields or super().get_exclude(request, obj)
+
+    def get_readonly_fields(self, request, obj=None):
+        fields = []
+
+        if not request.user.is_superuser:
+            fields.extend(["label"])
+
+        return fields or super().get_readonly_fields(request, obj)
```

Comment:
Based on the spreadsheet, only Admins should be able to see anything from `PemData`s: 

![image](https://github.com/user-attachments/assets/a8bd0fa5-caa1-4379-bde6-b2912780f79d)

So I don't think we should be specifying any fields as read-only. And then if all fields are just going to be excluded, it's probably better to not show the objects at all to non-superusers.

I suggest we replace this with an implementation of `has_view_permission` that makes explicit that non-superusers should not have view permission to this model.

Reply:
Thanks for the suggestion! I agree that it is more explicit and simpler to implement `has_view_permission` instead of using both `get_exclude` and `get_readonly_fields`.

~I also removed `list_display = ("label",)` from `class PemDataAdmin(admin.ModelAdmin)` since `PemData` already implements `__str__` in https://github.com/cal-itp/benefits/blob/main/benefits/core/models/common.py#L72, so it seemed redundant to still have it per the [docs](https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_display).~ Ah, yes, I agree about consistency, I'll make this quick change following https://github.com/cal-itp/benefits/pull/2673#discussion_r1953399793